postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario23- Update Amount/Payments - Confirm/request.json

```diff
@@ -29,12 +29,7 @@
       "key": "Accept",
       "value": "application/json"
     },
-    {
-      "key": "x-feature",
-      "value": "router-custom",
-      "type": "text",
-      "disabled": true
-    },
+    ,
```

Comment:
... and here.

Broke Postman tests for folks who had pulled the latest `main` branch.